### PR TITLE
[WIP] chore(readme): add structure for README file sections

### PR DESCRIPTION
**Pull request by JonLazariniDD**
add additional sections to the README file for testing, architecture, development standards and infrastructure.
Add meta on project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,90 @@
...
+
+
+## Architecture
+
+To capture and persist technical records for a Vehicle submitted by the client (VTM) which is then made available to VTA/VTM for searching a vehicle. If the submitted technical record is applicable for a ministry plate certificate, it triggers the generation of the certificate via dynamo streams of the technical results table.
+
+### End to end design
+[All in one view](https://wiki.dvsacloud.uk/pages/viewpage.action?pageId=79254695)
+
+![All in one](/docs/allinone.png)
```
Comment (akothiyal-deloitte) [inline README.md:12]: I think just having the link to confluence is sufficient. Including the image for the, all in one view isn't required, seems quite a lot in one picture, which isn't required for this specific microservice. Also, developer wont be updating the all in one view(architect would), so better single version of truth is maintained in confluence. All other diagrams on this readme are fine as they are specific to this microservice and being low level developers should be maintaining that.

File: README.md [hunk 2]
```diff
@@ -1,35 +1,90 @@
...
+
+              vin (Partition Key),
+
+              partialVin (Partition Key),
+
+#### Interactions/Integration with other AWS service
+The diagram illustrates that technical records integrate with Test Number service to request a unique number (systemNumber, trailorId, plateSerialNumber), before the information is saved into dynamo table. Technical record service also receives status updates from test result microservice to update a record from Provisional to Active [refer Test Result Submisson triggering Technical Record Status update](https://wiki.dvsacloud.uk/display/HVT/Test+Result+Submisson+triggering+Technical+Record+Status+update).
+![Integration AWS](/docs/awsintegration.png)
+
+
```
Comment (akothiyal-deloitte) [inline README.md:46]: include following lines here "Above Diagram shows orchestration/event publishing involved within Technical Records service. Following are the approved patterns it uses:"

File: README.md (hunk 1 above)
Comment (JonLazariniDD) [reply README.md:12]: good suggestion, I agree, doing the changes.
I found that a picture could make things look a bit better in the README and help a bit more.

File: README.md (hunk 2 above)
Comment (JonLazariniDD) [reply README.md:46]: done

**Pull request merged**